docs/release-notes/release-notes-0.5.0.md

```diff
@@ -0,0 +1,234 @@
+
+- A DEX server's host name and TLS certificates may now be updated. (<https://github.com/decred/dcrdex/pull/1605>, <https://github.com/decred/dcrdex/pull/1602>)
+
+- When a wallet regains peers after having lost connectivity, a notification is displayed. (<https://github.com/decred/dcrdex/pull/1701>, <https://github.com/decred/dcrdex/pull/1719>)
+
+- Reconfiguring BTC wallets is now less disruptive as it only requires an internal restart of the wallet and chain service when absolutely required. (<https://github.com/decred/dcrdex/pull/1686>)
+
+- The user interface files are now embedded in the "dexc" executable.  To use the DEX client, only the one file is required. Any "site" folders left from previous versions may be deleted. (<https://github.com/decred/dcrdex/pull/1710>)
+
+- The optional `dexcctl` command line control application now has a `version` command to ensure compatibility with dexc. (<https://github.com/decred/dcrdex/pull/1645>)
+
+NOTE: While Ethereum trading is not in production yet, the native ETH wallet can be used if built from source with the `-tags lgpl` switch.
+The built-in ETH wallet’s account private key can be exported, for import into other wallets like MetaMask or MEW. The ETH wallet is a “light” wallet that uses the native LES protocol.  Unfortunately, we are waiting on an update to the go-ethereum light client software to support The Merge.
+
+All the pieces for ETH and ERC20 trading are in place, however, The Merge upcoming in September requires the upstream light client package from go-ethereum to be updated before we can continue.  In addition The Merge creates a need for the full node for the server to run an additional consensus layer node on the server. The ETH release will hopefully be in several weeks.
+
+## Server changes
+
+- The server will check that a node's transaction indexes are enabled, when possible. (<https://github.com/decred/dcrdex/pull/1549>)
+
+- Fee rate estimation now has a median-based backup estimation method when the nodes fail to return a rate. (<https://github.com/decred/dcrdex/pull/1597>, <https://github.com/decred/dcrdex/pull/1655>)
+
+- The transaction search code is now smarter and blocks other routines less. (<https://github.com/decred/dcrdex/pull/1623>)
+
+- Confirmation checks are now throttled for fast blocks. (<https://github.com/decred/dcrdex/pull/1629>)
+
+- The server now begins signing untruncated messages, and ceases to recognize signatures created on truncated messages. (<https://github.com/decred/dcrdex/pull/1530>) This follows <https://github.com/decred/dcrdex/pull/1528> in v0.4.2 in which both client and server began recognizing signatures of the full messages, and <https://github.com/decred/dcrdex/pull/1529> from v0.4.3 in which clients began signing untruncated messages. The final change will be in v0.5.1 (<https://github.com/decred/dcrdex/pull/1526>), in which will cease to recognize the old signatures from servers that have not updated to v0.5.0.
```

Comment:
In Upgrading, will need to describe this minimum requirement of v0.4.2 clients for v0.5+ servers.